contracts/btc-finality/src/contract.rs

```diff
@@ -187,6 +193,11 @@ pub fn execute(
             &commitment,
             &signature,
         ),
+        ExecuteMsg::Jail {
+            fp_pubkey_hex,
+            duration,
+        } => handle_jail(deps, &env, &info, &fp_pubkey_hex, duration),
```

Comment:
Yeah I'd probably avoid this handler given that jailing shall not be bypassed in any way

Reply:
Do you mean, removing the ability of an admin to jail? We can remove it, yes.

Reply:
yep i don't see it's needed (for testing or for production)